app/lib/services/handle_submission_for_store.rb

```diff
@@ -33,6 +33,7 @@ def call
           works_in_school: store["works_in_school"] == "yes",
           employer_name: store["employer_name"].presence,
           employment_role: store["employment_role"].presence,
+          targeted_support_funding_eligibility: targeted_support_funding_eligibility,
```

Comment:
We're adding it to the DB here, should we also be adding it to the body sent over in Services::NpqProfileCreator in prep for ECF receiving this data? Or should we wait for the ECF side of things to define the API?